tensorflow_estimator/python/estimator/canned/linear.py

```diff
@@ -796,7 +796,7 @@ class LinearClassifierV2(estimator.EstimatorV2):
   estimator = LinearClassifier(
       feature_columns=[categorical_column_a,
                        categorical_feature_a_x_categorical_feature_b],
-      optimizer=tf.train.FtrlOptimizer(
+      optimizer=tf.keras.optimizers.Ftrl(
```

Comment:
Could you also update all the other doc strings with tf.train.FtrlOptimizer? Like the one in line 808, and the ones in LinearEstimatorV2 and LinearRegressorV2?

Reply:
+1.

Reply:
Made the changes downstream. Thanks!